File: seed/java-sdk/exhaustive/custom-dependency/src/main/java/com/snippets/Example29.java
package com.snippets;

import com.seed.exhaustive.SeedExhaustiveClient;

public class Example29 {
    public static void run() {
        SeedExhaustiveClient client = SeedExhaustiveClient
            .builder()
            .token("<token>")
            .url("https://api.fern.com")
            .build();

        client.endpoints().primitive().getAndReturnString("string");
    }
}

File: seed/csharp-sdk/exhaustive/explicit-namespaces/src/SeedApi.DynamicSnippets/Example29.cs
using global::System.Threading.Tasks;
using SeedExhaustive;

namespace Usage;

public class Example29
{
    public async global::System.Threading.Tasks.Task Do() {
        var client = new SeedExhaustive.SeedExhaustiveClient(
            token: "<token>",
            clientOptions: new SeedExhaustive.ClientOptions{
                BaseUrl = "https://api.fern.com"
            }
        );

        await client.Endpoints.Params.GetWithPathAsync(
            "param"
        );
    }

}
